deps/Makefile

```diff
@@ -32,6 +32,11 @@ include $(SRCDIR)/llvm-ver.make
 
 ## Some shared configuration options ##
 
+# don't pass makefile flags through by default
+# (which would often result in unintentionally overriding the flags we are trying to set here)
+MAKEOVERRIDES =
```

Comment:
I think I would also prefer we pass CFLAGS manually to dependencies. That way, the worse that can happen is a build failure, which doesn't go unnoticed. OTOH, when setting `MAKEOVERRIDES`, dependencies will silently not use the distribution CFLAGS, which can give more obscure issues (like ABI discrepancies on ARM, missing security-related flags, etc.).

Anyway most distributions only need to build libuv, and maybe libunwind until upstream makes a new release, so we can just fix these two cases for now.